### PR TITLE
Update hsts.md

### PR DESCRIPTION
**Pull request by ari-gold**
Closes #

## Effect
PR includes the following changes:
-
-

## Remaining Work
- [ ] List any outstanding work here
- [ ] Technical review
- [ ] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Archive from **Done** in [Main Project](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by alexfornuto** [comment]
@ari-gold when you're ready you can remove the label `In Progress` and add `Technical Review`, and unassign yourself.

File: source/partials/hsts.md [hunk 1]
```diff
@@ -4,33 +4,29 @@ Before adjusting `enforce_https`, review and understand the configuration option
 
 </Alert>
 
-Ensure that your site will always use HTTPS to deliver content with `enforce_https`. Five values are available, from least to most secure:
 
-Use of the HSTS header is defined by the `enforce_https` directive, and takes five possible values which are handled by Pantheon as shown below:
-
-|       enforce_https:                                          | Redirect                           |   HSTS   | Strict-Transport-Security | includeSubdomains                  | preload                            |
-|:-------------------------------------------------------------:|:----------------------------------:|:--------:|---------------------------|:----------------------------------:|:----------------------------------:|
-| `off` (default)                                               |     ❌                             | Disabled | Not set by Pantheon       |         ❌                         |    ❌                              |
```
Comment (alexfornuto) [inline source/partials/hsts.md]: @ari-gold is `off` no longer an accepted value?

**Comment by alexfornuto** [comment]
@ari-gold are you still working on this, or should we consider it outdated by #5168?

**Comment by ari-gold** [comment]
Yes, I need to rebase and finalize

**Comment by carolyn-shannon** [comment]
@ari-gold any updates on this?

**Comment by carolyn-shannon** [comment]
Followed up, waiting for feedback from @ari-gold.

**Comment by ari-gold** [comment]
I'll carry this forward in a different PR.

**Pull request closed without merging**